### PR TITLE
Moved the UI documentation

### PR DESCRIPTION
**Pull request by kuby**

### CODE REVIEW

File: docs/guide/developing/dashboard.md [hunk 1]
```diff
@@ -1,65 +1,91 @@
...
+
+### Requirements
+
+```bash
+nodejs
+```
+
+### Configuration
+
+Configuration for the client can be found in `./src/auth_config.json`.
```
Comment (ondrejtomcik) [inline docs/guide/developing/dashboard.md]: `./http-gateway/...`

File: docs/guide/developing/dashboard.md [hunk 2]
```diff
@@ -1,65 +1,91 @@
...
+
+```json
+{
+  "domain": "",
```
Comment (ondrejtomcik) [inline docs/guide/developing/dashboard.md]: Can you provide examples? For example same as we have for try.plgd.cloud

File: docs/guide/developing/dashboard.md [hunk 3]
```diff
@@ -1,65 +1,91 @@
...
+$ npm run generate-language-files
+```
+
+Now your translations are updated and ready to be used.
+
+## Branding
+
+### Colors change
+
+All colors are defined in one `scss` file located in `/src/common/styles/colors.scss`. Changing one of the colors will have an impact on all parts of the application.
```
Comment (ondrejtomcik) [inline docs/guide/developing/dashboard.md]: ./http-gateway/....

File: docs/guide/developing/dashboard.md (hunk 1 above)
Comment (kuby) [reply docs/guide/developing/dashboard.md]: Added full path.

File: docs/guide/developing/dashboard.md (hunk 3 above)
Comment (kuby) [reply docs/guide/developing/dashboard.md]: Added full path.

File: docs/guide/developing/dashboard.md (hunk 2 above)
Comment (kuby) [reply docs/guide/developing/dashboard.md]: Provided example.

**Review by ondrejtomcik — APPROVED** [review]

**Pull request merged**